packages/sonarwhal/tests/lib/utils/rule-helpers.ts

```diff
@@ -143,15 +143,15 @@ test.serial(`getRuleName - returns an empty string if it can't determine the rul
     t.deepEqual(ruleName, '');
 });
 
-test.serial(`getRuleName - returns a combined string if a package name is passed`, (t) => {
-    const sandbox = sinon.createSandbox();
+// test.serial(`getRuleName - returns a combined string if a package name is passed`, (t) => {
```

Comment:
uncomment or remove it :)

Reply:
I have to rewrite all the `getRuleName` tests. None work after this and some of the previous logic didn't make any sense.